ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -139,7 +139,19 @@ pipeline {
         }
         cleanup {
           script {
-            sh "python3 ./ci-scripts/generateHtmlReport-OAI-pipeline.py --mode=Build --job_name=${JOB_NAME} --job_id=${BUILD_ID} --job_url=${BUILD_URL} --git_url=${GIT_URL} --git_src_branch=${GIT_BRANCH} --git_src_commit=${GIT_COMMIT}"
+            if (env.ghprbPullId != null) {
+              commitID = sh returnStdout: true, script: 'git rev-parse origin/master'
```

Comment:
Don't we have the commitID with the Git plugin in an env variable? something like `env.GIT_COMMIT`

Reply:
In fact in the previous run, when it is a pull request, that `env.GIT_COMMIT` is `null`.

That's why I use the `ghprb` variables.